bin/nanocld/changelog.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [untagged] - yyyy-mm-dd
+
+### Added
+
+- Endpoint `GET /process/{name}/stats` to get process stats by it's name by [n0tank3sh](https://github.com/n0tank3sh)
+
+### Fixed
```

Comment:
You can remove this part, i have merge it with 0.16.3 release and already deployed it

Reply:
Fixed it